containers/2_ApplicationSpecific/fds/README.md

```diff
@@ -1,45 +1,69 @@
+
+This container includes Intel MPI Library 2021.6.  CCR's `ccrsoft/2023.01` [software environment](https://docs.ccr.buffalo.edu/en/latest/software/releases/#202301) includes a module for `intel/2022.00` which includes `impi 2021.5.0`  These two versions of MPI are alike enough that this container works when loading the `intel/2022.00` module.  However, the user should be aware with future versions of FDS and CCR's intel modules, that you must align MPI versions on the system with the container in order for them to work properly.  See [here](https://docs.ccr.buffalo.edu/en/latest/howto/containerization/#building-mpi-enabled-images) for more info.
 
 ## Pulling the container  
 
-Please refer to CCR's [container documentation](https://docs.ccr.buffalo.edu/en/latest/howto/containerization/) for information on using Apptainer.  This example shows how to pull a container from Docker Hub and convert it to an Apptainer image file.  This can be done from a [compile node or a compute node](https://docs.ccr.buffalo.edu/en/latest/hpc/clusters/#node-types).  Apptainer is not available on the CCR login nodes.  
+The FDS software can be pulled from Docker Hub to the CCR's HPC environment using Apptainer. This process follows the same steps detailed in the [introductory container example](../../0_Introductory/README.md#pulling-the-container), which you can use as a guide. Please refer to CCR's [container documentation](https://docs.ccr.buffalo.edu/en/latest/howto/containerization/) for more information on using Apptainer.
+
+Log into a compute node, navigate to your project directory, and set a temporary directory for cache there.
+
+Once ready, pull the FDS container from Docker Hub:
 
 ```
-CCRusername@login:~$ ssh compile  
-CCRusername@compile:~$ cd /projects/academic/[YourGroupName]/[CCRusername]  
-CCRusername@compile:~$ mkdir cache  
-CCRusername@compile:~$ export APPTAINER_CACHEDIR=/projects/academic/[YourGroupName]/[CCRusername]/cache
-CCRusername@compile:~$ apptainer pull docker://satcomx00/fds:6.7.9
+export APPTAINER_CACHEDIR=/projects/academic/[YourGroupName]/[CCRusername]/cache
+apptainer pull docker://satcomx00/fds:6.7.9
 ```
-This will save the Apptainer image as `fds_6.7.9.sif` in your current working directory.  
+
+After the pull completes, the Apptainer image will be saved as `fds_6.7.9.sif` in your current working directory.  
 
 
 ## Running the container image  
 
-You can run FDS either in an interactive job or non-interactively by using a batch script (recommended).  
+You can run FDS either in an interactive job or non-interactively by using a batch script (recommended).
+
+The following information on running FDS is provided as an example only and not all users will have access to the resources in this example.  The [`radiator.fds`](radiator.fds) input file is taken from the FDS website and used as an example.  This will run on, at most, 3 tasks.  Your real world work may scale to more tasks.
```

Comment:
```suggestion
The following information on running FDS is provided as an example only.  The [`radiator.fds`](radiator.fds) input file is taken from the FDS website and used as an example.  This will run on, at most, 3 tasks.  Your real world work may scale to more tasks.
```
Removed that wording as it was in reference to the salloc options but that has been removed.